ui/src/components/common/SummaryPageLayout/index.tsx

```diff
@@ -301,6 +305,7 @@ export function SummaryPageLayout({
   ]);
 
   const contentMargin = useMemo(() => {
+    if (collapsable) return 0;
```

Comment:
Confused about this change. We want "contentMargin" to be the number of px to push the main content down so the summary isn't overlaying it. 

Wouldn't this always return 0 for the pipeline's usage, given it is "collapsable"? 

Looking further into the changes, I see what you are doing. I think it would be best to have SummaryPageLayout still apply the marginTop as it did before. If you want it to not apply any margin when it collapses, just pass `offsetOnCollapse` as false. 

Adding it to context to adjust your other items (buttons and etc.) is fine, but i think the main layout margin should be controlled here.